src/main/java/me/totalfreedom/totalfreedommod/command/Command_loginmessage.java

```diff
@@ -10,7 +10,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.OP, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Change your login message", usage = "/<command> [message]")
+@CommandParameters(description = "Change your login message", usage = "/<command> <message>")
```

Comment:
This should stay as optional - `/loginmessage` will open up the premade login messages (in an inventory)